plugins/linux-capture/pipewire.c

```diff
@@ -116,6 +126,32 @@ struct dbus_call_data {
 
 /* auxiliary methods */
 
+static bool parse_pw_version(struct obs_pw_version *dst, const char *version)
+{
+	int n_matches = sscanf(version, "%d.%d.%d", &dst->major, &dst->minor,
+			       &dst->micro);
+	return n_matches == 3;
+}
+
+static bool check_pw_version(const struct obs_pw_version *pw_version, int major,
+			     int minor, int micro)
+{
+	return pw_version->major >= major && pw_version->minor >= minor &&
+	       pw_version->micro >= micro;
+}
+
+static void update_pw_versions(obs_pipewire_data *obs_pw, const char *version)
+{
+	blog(LOG_INFO, "[pipewire] server_version %s", version);
+	blog(LOG_INFO, "[pipewire] library_version %s",
+	     pw_get_library_version());
+	blog(LOG_INFO, "[pipewire] header_version %s",
+	     pw_get_headers_version());
```

Comment:
Nitpicks: please remove the `_` and add a `:` to the debug messages, e.g.: `[pipewire] server version: %s`